CHANGES.md

```diff
@@ -10,6 +10,18 @@ To be released.
 
  -  `BaseStore` class became to implement `IDisposable`.  [[#789]]
  -  Removed `IStore.DeleteIndex(Guid, HashDigest<SHA256>)` method.  [[#802]]
+ -  Extension classes have been renamed. However, it is not affected
+    if you do not call it directly.  [[#803]]
```

Comment:
```suggestion
 -  Extension classes was renamed. However, it would not be affected
    if you have called it by using instance method syntax.  [[#803]]
```

Reply:
This seems reverted?

Reply:
My mistake. Restored it.